.github/workflows/kind.yml

```diff
@@ -174,6 +174,61 @@ jobs:
           path: log.tar.gz
           retention-days: 30
 
+  test-e2e-multicluster:
+    name: E2e tests on a Kind cluster for multi-cluster feature on Linux
+    needs:
+      - build-clickhouse-monitor-image
+      - build-clickhouse-server-image
+    runs-on: [ubuntu-latest]
+    steps:
+      - name: Free disk space
+        # https://github.com/actions/virtual-environments/issues/709
+        run: |
+          sudo apt-get clean
+          sudo rm -rf /usr/share/dotnet
+          sudo rm -rf /opt/ghc
+          sudo rm -rf "/usr/local/share/boost"
+          sudo rm -rf "$AGENT_TOOLSDIRECTORY"
+          sudo rm -rf "/usr/local/lib/android"
+          df -h
+      - uses: actions/checkout@v3
+      - uses: actions/setup-go@v4
+        with:
+          go-version-file: 'go.mod'
+      - name: Download ClickHouse monitor images from previous jobs
+        uses: actions/download-artifact@v3
+        with:
+          name: clickhouse-monitor
+      - name: Download ClickHouse server images from previous jobs
+        uses: actions/download-artifact@v3
+        with:
+          name: clickhouse-server
+      - name: Load Theia image
+        run:  |
+          docker load -i clickhouse-monitor.tar
+          docker tag antrea/theia-clickhouse-monitor:latest projects.registry.vmware.com/antrea/theia-clickhouse-monitor:latest
+          docker load -i clickhouse-server.tar
+          docker tag antrea/theia-clickhouse-server:latest projects.registry.vmware.com/antrea/theia-clickhouse-server:latest
+      - name: Install Kind
+        run: |
+          curl -Lo ./kind https://github.com/kubernetes-sigs/kind/releases/download/${KIND_VERSION}/kind-$(uname)-amd64
+          chmod +x ./kind
+          sudo mv kind /usr/local/bin
+      - name: Run e2e tests for multi-cluster feature
+        run: |
+          mkdir log
+          ANTREA_LOG_DIR=$PWD/log ./ci/kind/test-multi-cluster.sh
+      - name: Tar log files
```

Comment:
In general, coverage isn't being collected/reported for multicluster e2e tests, can we add a Codecov coverage section here with the multicluster-e2e flag?